ci/test_model.py

```diff
@@ -6,37 +9,43 @@
 
 
 def _asr(model_name):
-    d = ModelDownloader()
-    speech2text = Speech2Text(**d.download_and_unpack(model_name))
+    d = ModelDownloader("downloads")
+    speech2text = Speech2Text(**d.download_and_unpack(model_name, quiet=True))
     speech = np.zeros((10000,), dtype=np.float32)
     nbests = speech2text(speech)
     text, *_ = nbests[0]
     assert isinstance(text, str)
 
 
 def _tts(model_name):
-    d = ModelDownloader()
-    text2speech = Text2Speech(**d.download_and_unpack(model_name))
-    speech = np.zeros((10000,), dtype=np.float32)
+    d = ModelDownloader("downloads")
+    text2speech = Text2Speech(**d.download_and_unpack(model_name, quiet=True))
+    inputs = {"text": "foo"}
     if text2speech.use_speech:
-        text2speech("foo", speech=speech)
-    else:
-        text2speech("foo")
+        inputs["speech"] = np.zeros((10000,), dtype=np.float32)
+    if text2speech.tts.spk_embed_dim is not None:
+        inputs["spembs"] = np.zeros((text2speech.tts.spk_embed_dim,), dtype=np.float32)
+    text2speech(**inputs)
 
 
 def test_model():
     d = ModelDownloader()
     tasks = ["asr", "tts"]
 
     for task in tasks:
-        for model_name in d.query(task=task):
-            if d.query("valid", name=model_name)[0] == "false":
-                continue
-            print(f"#### Test {model_name} ####")
-
-            if task == "asr":
-                _asr(model_name)
-            elif task == "tts":
-                _tts(model_name)
-            else:
-                raise NotImplementedError(f"task={task}")
+        for corpus in list(set(d.query("corpus", task=task))):
+            for model_name in d.query(task=task, corpus=corpus):
+                if d.query("valid", name=model_name)[0] == "false":
+                    continue
+                print(f"#### Test {model_name} ####")
+
+                if task == "asr":
+                    _asr(model_name)
+                elif task == "tts":
+                    _tts(model_name)
+                else:
+                    raise NotImplementedError(f"task={task}")
+
+            # NOTE(kan-bayashi): remove and recreate cache dir to reduce the disk usage.
+            shutil.rmtree("downloads")
```

Comment:
I remembered that I also created a method to empty cache directory.

https://github.com/espnet/espnet_model_zoo/blob/80b053a6ee6259ff09d0e017a2ca85106d327c05/espnet_model_zoo/downloader.py#L110-L113

This is a note.